src/InteractiveWindow/VisualStudio/source.extension.vsixmanifest

```diff
@@ -4,6 +4,7 @@
     <Identity Id="1F42C6D0-F876-4AF0-8185-1BEB0A325BB2" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>VisualStudio Interactive Components</DisplayName>
     <Description>Interactive components for Visual Studio.</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\VisualStudioInteractiveComponents</ShortcutPath>
```

Comment:
These are the bits used by non-ML too, right? Should this just be CommonExtensions\Microsoft\InteractiveWindow?